test/controllers/sign.out.controller.spec.ts

```diff
@@ -71,8 +73,8 @@ describe("Sign Out controller", () => {
 
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(SIGN_OUT_PAGE_TITLE);
-      expect(resp.text).toContain(SIGN_OUT_HINT_TEXT);
-      expect(resp.text).toContain(SIGN_OUT_HELP_DETAILS_TEXT);
+      expect(resp.text).toContain(SIGN_OUT_SAVE_AND_RESUME_GUIDANCE_TEXT);
```

Comment:
I wasn't expecting these two lines to change, rather that line 79 was modified to reference the new constant and then a new 'expect' line added below with the second constant.